client/my-sites/plan-features/style.scss

```diff
@@ -103,7 +103,8 @@ $plan-features-sidebar-width: 272px;
 
 	@include breakpoint( '<1040px' ) {
 		border-spacing: 0;
-		margin: 0 15px;
+		margin-left: 15px;
+		margin-right: 15px;
```

Comment:
This is the same as the current code, just not shorthand?

Reply:
Using shorthand syntax, all 4 margin props (top, right, bottom, left) are updated. Due to which `margin-bottom` applied via different css ruleset is overridden as per screenshot below

![Plan options UI issue](https://user-images.githubusercontent.com/11641900/66532962-e08ef900-eb2e-11e9-8f01-00061925c651.png)

Hence only the target margin props (`margin-left`, `margin-right`) are applied, preserving `margin-bottom` value. @apeatling

Reply:
Got it, thanks for clarifying.